chapter8.adoc

```diff
@@ -1,37 +1,97 @@
+`msdcfg.sdedbgalw`.
+
+When M-mode external debug is disabled, whether execution at privilege modes
+less than `M-mode` may be debugged by an external debugger depends on the
+configuration held in `msdcfg.sdedbgalw`, as described below:
+
+When `msdcfg.sdedbgalw` is 0:
+
+* Access by external debuggers to the memory and/or state of the supervisor domain is disallowed.
+* Entry to Debug Mode from a supervisor domain is disallowed.
+
+
+When `msdcfg.sdedbgalw` = 1 then external debug of privilege modes less than
+`M-mode` is allowed for such a supervisor domain on a per-hart basis, 
+with the additional requirements listed below. +
+
+To enforce the above controls specified by this extension, the following
+functional and security requirements must be met by the external secure debug
+system cite:[ExtDbgSec] to meet the security objectives of supervisor domain
+isolation:
+
+. External debug of M-mode can be enabled only by the HW RoT of the RDSM
+ (See RISC-V Security model requirements SR_GEN_007 and SR_GEN_012).
+. External debug must be able to access supervisor domain memory and/or state. 
+  In this context, "state" includes all supervisor domain resources accessible per the 
+  Debug specification cite:[ExtDbg].
+. Entry to Debug Mode from a supervisor domain is allowed.
+. The RDSM must be able to context switch all triggers even when `d-mode` = 1.
+  This functional change allows the RDSM to remain in control of external debug
+  for supervisor domains when it is not under external debug itself.
+
+=== `Smsdetrc`: External Trace allowed control for Supervisor Domain
+
+When M-mode external trace is enabled, all supervisor domains activity may also
+be traced by an external trace tool irrespective of the configuration held in
+`msdcfg.sdetralw`.
+
+When M-mode external trace is disabled, whether execution at privilege modes
+less than `M-mode` may be traced by an external trace tool depends on the
+configuration held in `msdcfg.sdetrcalw`, as described below:
+
+When `msdcfg.sdetrcalw` = 0, external trace of the supervisor domain is disallowed.
+
+When `msdcfg.sdetrcalw` = 1 then external trace of privilege modes less than
+`M-mode` shall be allowed for the SD on a per hart basis, with the
+additional requirements listed below.
+
+To enforce the above controls specified by this extension, the following
+functional and security requirements must be met by the external secure debug
+system cite:[ExtDbgSec] to meet the security objectives of supervisor domain
+isolation:
+
+. External trace of M-mode can be enabled only by the HW RoT of the RDSM
+ (See RISC-V Security model requirements SR_GEN_007 and SR_GEN_012).
+. When external tracing is disabled, tracing must be stopped on:
+.. entry to Debug mode
+.. entry to `M-mode` privilege
+.. entry to privilege less than `M-mode` with `msdcfg.sdetrcalw` = 0.
+. When external tracing is disabled, tracing must be started on:
```

Comment:
Is `disabled` a typo? 
However, even if it intends to mean `enabled` here, the tracing does not necessarily start because it is additional controlled by `trTeEnable`

Reply:
actually, that second point is not needed any more. removing it.